docs/rules/template-tag-spacing.md

```diff
@@ -0,0 +1,78 @@
+# Enforce Usage of Spacing in Tagged Template Literals (template-tag-spacing)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+With ES6, it's possible to create functions called [tagged template literals](#further-reading) where the function parameters consist of a template literal's strings and expressions.
+
+When using tagged template literals, it's possible to insert whitespace between the tag function and the template literal. Since this whitespace is optional, the following lines are equivalent:
+
+```js
+let hello = func`Hello world`;
+let hello = func `Hello world`;
+```
+
+## Rule Details
+
+This rule aims to maintain consistency around the spacing between template tag functions and their template literals.
+
+## Options
+
+```json
+{
+    "template-tag-spacing": ["error", "never"]
+}
+```
+
+This rule has one option which has either `"never"` or `"always"` as value.
```

Comment:
Looks like there's a word missing here - maybe `This rule has one option whose value can be set to "never" or "always"`?